serverlessworkflow/modules/ROOT/pages/tooling/serverless-logic-web-tools/serverless-logic-web-tools-openshift-integration.adoc

```diff
@@ -23,6 +23,11 @@ A window appears containing your current operating system as selected and a link
 --
 You might need to grant permissions to run the file depending on your operating system and settings.
 
+[NOTE]
+====
+If you are using macOS, you might encounter the `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is due to the overeage https://support.apple.com/en-us/HT202491[approach towards security] by macOS and you can fix it by moving the "KIE Sandbox Extended Services" application out of quarantine. To achieve this, you can execute the `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app` command. 
```

Comment:
I am assuming the word is overage.
```suggestion
If you are using macOS, you might encounter the `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is due to the overage https://support.apple.com/en-us/HT202491[approach towards security] by macOS and you can fix it by moving the "KIE Sandbox Extended Services" application out of quarantine. You can execute the `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app` command to achieve this.  
```

Reply:
Sorry, it was supposed to be "overeager".